code/modules/hydroponics/grown/towercap.dm

```diff
@@ -268,12 +268,19 @@
 /obj/structure/bonfire/proc/Cook()
 	var/turf/current_location = get_turf(src)
 	for(var/A in current_location)
+		var/obj/G = A;
 		if(A == src)
 			continue
 		else if(isliving(A)) //It's still a fire, idiot.
 			var/mob/living/L = A
 			L.adjust_fire_stacks(fire_stack_strength)
 			L.IgniteMob()
+		else if(G.GetComponent(/datum/component/grillable))
+			if(SEND_SIGNAL(G, COMSIG_ITEM_GRILLED, src) & COMPONENT_HANDLED_GRILLING)
+				continue
+			G.fire_act(1000) //Hot hot hot!
+			if(prob(10))
+				visible_message("<span class='danger'>[G] doesn't seem to be doing too great on the [src]!</span>")
```

Comment:
```suggestion
				visible_message("<span class='danger'>[G] doesn't seem to be doing too great on [src]!</span>")
```

Reply:
"doesn't seem to be doing too great on the bonfire" vs "doesn't seem to be doing great on bonfire"?